ui/partials/main/thWatchedRepoNavPanel.html

```diff
@@ -42,7 +42,7 @@
 
         <!--Hidden Jobs Button-->
         <span class="btn btn-view-nav btn-sm btn-right-navbar"
-              title="Show/hide excluded jobs"
+              title="{{ isSkippingExclusionProfiles ? 'Show excluded jobs' : 'Hide excluded jobs'}}"
```

Comment:
Somewhat counter-intuitively, this should be `!isSkippingExclusionProfiles`, since "skipping exclusion profiles" == "show excluded jobs"